### ISSUE TITLE
Multiple include problem

### ISSUE THREAD

**Issue by goschevski** [mozilla/nunjucks#492]
Hi,

I'm using browserify and [nunjucksify](https://www.npmjs.com/package/nunjucksify) to pre-compile templates.

I have a problem when I want to add 2 or more `include` statements in the template

> layouts/base.nunj

```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1" />

        <title></title>
        <meta name="description" content="" />

        <link rel="stylesheet" href="public/css/style.css">
    </head>
    <body>
        {% block page %}{% endblock %}

        <script src="public/js/index.js"></script>
    </body>
</html>
```

> pages/home.nunj

```
{% extends "../layouts/base.nunj" %}

{% block page %}
    <div class="Page">
        {% include "../partials/shared/header.nunj" %}

        {% include "../partials/home/content.nunj" %}

        {% include "../partials/shared/footer.nunj" %}
    </div>
{% endblock %}
```

If I leave only 1 of those 3 partials, it will work. If I have 2 or more included partials i get this error:

```
Uncaught Template render error: (unknown path)
Template render error: Template render error: Template render error: (unknown path)
RangeError: Maximum call stack size exceeded
```

Am I doing something wrong or this is a bug?

**Comment by jlongster** [comment]
It may be a problem with nunjucksify, or something that was fixed in the latest 2.0. I cannot reproduce. Please re-open if you can still reproduce without nunjucksify.

**Comment by goschevski** [comment]
Thanks, I've asked for help on nunjucksify repo - https://github.com/rotundasoftware/nunjucksify/issues/10
